classes/activity-manager.js

```diff
@@ -146,7 +146,7 @@ class ActivityManager {
      * @private
      */
     static parseRole(member, role, activityName) {
-        let stampNumber = parseInt(role.name.substring(role.name.length - 2));
+        let stampNumber = parseInt(role.name.split('#')[1]);
```

Comment:
I like this cuz its fast, but it requires the user to keep to a consistent naming convention. Could we instead look at the current role ID, find it within our stampRole collection, get the key "stamp number" for that role and then give the user the next role?.

We could just stick with having a naming convention, this also affects how we search for stamp roles before calling parseRole.

If we don't want a naming convention, we can also search through the user's roles, find a role ID that can be found in the stampRoles and then proceed from there?

Thinking about this, it feels like we will be doing a roleID search more often that a role number search, might be better to change the collection from <stamp number, role ID> to <Role ID, stamp number>.

Thoughts?